cli/src/config/templates.toml

```diff
@@ -91,7 +91,7 @@ label(if(overridden, "overridden"),
 
 builtin_draft_commit_description = '''
 concat(
-  coalesce(description, "\n"),
+  coalesce(description, default_commit_description),
```

Comment:
nit: I would leave `"\n"` here as `coalesce(description, default_commit_description, "\n")` because it's not so obvious why the default is `"\n"`.

Reply:
Done. I also made the default description an empty string.